src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -62,7 +65,7 @@
     /**
      * A lot of luck to get a guaranteed saplings drop.
      */
-    private static final int A_LOT_OF_LUCK = 100;
+    private static final int A_LOT_OF_LUCK = 10000;
 
```

Comment:
That's a ton of luck =P

Reply:
the more luck the less rolls trying to get a drop! :D